pyQuARC/schemas/rule_mapping.json

```diff
@@ -4020,6 +4087,18 @@
                         "DataCenters/ContactInformation/RelatedUrls"
                     ]
                 }
+            ],
+            "dif10": [
+                {
+                    "fields": [
+                        "DIF/Multimedia_Sample/URL"
```

Comment:
Should this be DIF/Multimedia_Sample? 
https://git.earthdata.nasa.gov/projects/EMFD/repos/dif-schemas/browse/10.x/dif_v10.3.xsd#1754

Reply:
I agree that it should. Fixed this in the most recent commit